invenio_app_rdm/administration/records/records.py

```diff
@@ -21,11 +22,11 @@ class RecordAdminListView(AdminResourceListView):
     """Configuration for OAI-PMH sets list view."""
 
     api_endpoint = "/records"
-    name = "Records"
+    name = "records"
```

Comment:
why do you change that? i am not sure if the name is not used somewhere to reference this view. did you tested that?

Reply:
I double-checked with other references and looks like it is the right way to be lowercase see:
[moderation](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/administration/user_moderation/user_moderation.py#L25)
[users](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/administration/users/users.py#L54)
[domains](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/administration/domains/domains.py#L40)
I also checked in other packages see:
[banners](https://github.com/inveniosoftware/invenio-banners/blob/master/invenio_banners/administration/banners.py#L23)

Reply:
I will revert this change on this PR to make it only for I18n.

Reply:
Changes not related to i18n is been moved to this [PR](https://github.com/inveniosoftware/invenio-app-rdm/pull/2887)